### ISSUE TITLE
TabView: Accessibility - tablist role should be accessibility parent of elements with tab role

### ISSUE THREAD

**Issue by DanRoddis** [primefaces/primevue#6229]
### Describe the bug

The tab role should be a direct child of the tablist role to maintain the correct relationship and ensure that assistive technologies can properly interpret and navigate the structure. When the tab elements are not direct descendants of the tablist, screen readers and other assistive tools might not correctly associate the tabs with the tablist, which can lead to an accessibility failure.

> Authors MUST ensure [elements](https://dom.spec.whatwg.org/#concept-element) with [role](https://w3c.github.io/aria/#dfn-role) [tab](https://w3c.github.io/aria/#tab) are [accessibility children](https://w3c.github.io/aria/#dfn-accessibility-child) of an element with the role [tablist](https://w3c.github.io/aria/#tablist).

The specific wording is 'children' when the ARIA spec uses the word 'children' to mean direct descendants and 'descendants' when this direct relationship is not necessary. When viewing the accessibility tree the `<ul>` element is between the tablist and the tab role

![image](https://github.com/user-attachments/assets/26e726b8-75f4-45df-85fe-17807498cdd4)

### Reproducer

https://stackblitz.com/edit/2pvw6d?file=src%2FApp.vue

### PrimeVue version

3.53.0

### Vue version

4.x

### Language

TypeScript

### Build / Runtime

Vue CLI App

### Steps to reproduce the behavior

Issue is evident running automated accessibility tool against the tab example stackblitz

### Expected behavior

Component does not cause accessibility warnings, tablist role is on the direct parent of tabs

**Comment by tugcekucukoglu** [comment]
We could suggest using the new Tabs component. If you think you've found the same problem, please create a new issue.
